docs/medical-api/api-reference/document/start-document-query.mdx

```diff
@@ -7,8 +7,10 @@ api: "POST /medical/v1/document/query"
 When executed, this endpoint triggers an asynchronous document query with HIEs and immediately returns the status
 of document query, `processing`.
 
+As the query is processing if you were to trigger this enpoint again you will receive that the query status is `processing` as well as the current progress of documents being queried.
```

Comment:
nit: enpoint

nit: adding some commas would be good: "As the query is processing, if you were to trigger this endpoint again, you will receive that the query status is `processing` as well as the current progress of documents being queried."